### ISSUE TITLE
Modifying the storage directory for the shared-file disk template doesn't work

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1157]
Originally reported of Google Code with ID 1101.

```
<b>What steps will reproduce the problem?</b>
1. Create a cluster
2. Issue

    gnt-cluster modify --shared-file-storage-dir /foo/bar

3. Run

    gnt-cluster info | grep shared.file.storage

<b>What is the expected output? What do you see instead?</b>

The new value isn't used at all, even though the job reports as successful.

Looks like an omission in the code that'll be easy to fix.
```
Originally added on 2015-06-09 15:37:02 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in

commit b5f3cca8c663c626e3209fc05f510037d9d4fdb4
Author: Petr Pudlak <pudlak@google.com>
Date:   Wed Jun 10 11:10:01 2015 +0200

    Fix tests for setting (shared) file storage directory
    
    - Fix the test for setting file_storage_dir, which didn't check if the
      value was really set.
    - Add tests for shared_file_storage_dir, which were missing completely.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

commit 5d7d31d174a73b58494687e2c9132bbf2f3002f0
Author: Petr Pudlak <pudlak@google.com>
Date:   Wed Jun 10 11:20:35 2015 +0200

    Add missing call for setting shared file storage directory
    
    With the call missing, it wasn't possible to change the directory after
    cluster initialization. Fixes #1101.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2015-06-10 11:33:10 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-07-13 17:13:16 +0000 UTC.

Changed State: Released
